spec/activerecord-bitemporal/bitemporal_spec.rb

```diff
@@ -2289,6 +2289,143 @@ class EmployeeWithUniquness < Employee
       end
     end
 
+    describe "ActiveRecord::Bitemporal.bitemporal_at" do
+      context "instance object" do
+        let!(:company) { Company.create(name: "Company") }
+        before do
+          emp = company.employees.create(name: "Jane")
+          emp.address = Address.create(name: "Address")
```

Comment:
Wouldn't it be better to use `create_address` ?
In the current code, the `address.employee_id` is updated after the Address is created.
```suggestion
          emp.create_address(name: "Address")
```

Reply:
🤦 You're right.

Reply:
Fixed https://github.com/kufu/activerecord-bitemporal/compare/8660a03f19d2f7c09d25ecd81175f2f24eaa8114..7d7cb6df0757eb1950ca0d3d7506f999e2316aae